src/api/static-banner/content-types/static-banner/ifecycles.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = {
+  async beforeCreate(event) {
+    await validateImage(event);
+  },
+  async beforeUpdate(event) {
+    await validateImage(event);
+  },
+};
+
+async function validateImage(event) {
+  const { data } = event.params;
+
+  if (data.image) {
+    const media = await strapi.db
+      .query("plugin::upload.file")
+      .findOne({ where: { id: data.image } });
+
+    if (media.width !== 1200 && media.height !== 400) {
+      throw new Error("Incorrect image dimensions. Required dimensions are 1200x400.");
+    }
```

Comment:
why you adding code here?

Reply:
Validar las dimensiones de la imagen y garantizar la calidad de la imagen.